tests/scattering2d/test_torch_scattering2d.py

```diff
@@ -236,10 +233,6 @@ def test_cdgmm_exceptions(self, backend):
             backend.cdgmm(torch.empty(3, 4, 5, 2), torch.empty(4, 3, 2))
         assert 'not compatible' in exc.value.args[0]
 
-        with pytest.raises(TypeError) as exc:
```

Comment:
Why are we no longer testing for this?

Reply:
What if we do multiplication with a real filter and a real input? that should pass, shouldn't it?

Reply:
OK, good point.

Reply:
has this been taken care of?

Reply:
has this been taken care of?

Reply:
yes